README.md

```diff
@@ -4,7 +4,7 @@
 [![Boxtron Discord](https://img.shields.io/discord/514567252864008206.svg?label=discord)](https://discord.gg/8mFhUPX)
 [![Say Thanks!](https://img.shields.io/badge/Say%20Thanks-!-1EAEDB.svg)](https://saythanks.io/to/dreamer)
 
-Compatibility tool to run DOS games on Steam through native Linux DOSBox.
+Steam Play compatibility tool to run DOS games on Steam through native Linux DOSBox.
```

Comment:
"Steam Play compatibility tool to run DOS games on Steam (…)" is redundant.

Also, amend the commit message please :) this project is starting to stabilize and I prefer to up the standards a little bit; here's guideline how to write commit message: https://chris.beams.io/posts/git-commit/

Reply:
You mean the "Steam" part in the end? Good point about the commit though, let me change the description

Reply:
Changing title does not change the commit message - they are separate things :)

You need to `git commit --amend` in your local repo, edit commit message and `git push --force <your-remote-name> patch-1` (this will overwrite last commit in GitHub and automatically update commit in review).

Reply:
Ah crap, guess I'll add it to another thing you can't do though GitHub browser. I kinda like using it out of convinience for typos, but guess I'll have to stop being lazy. Either way, other than commit message, what should the actual text be? `Steam Play compatibility tool to run DOS games through native Linux DOSBox.`? If so, I'll adjust it and do the same for dreamer/luxtorpeda#39

Reply:
Yeah, that sounds better. You can also just push a new commit to the review - I can squash them together and update commit message once review will be done.